### ISSUE TITLE
Don't show indentation or comas in diff

### ISSUE THREAD

**Issue by erdnaxeli** [hjwp/pytest-icdiff#24]
The diff shows indent and coma diff, which useless.

![image](https://user-images.githubusercontent.com/479441/160856409-d6cc4dcf-9a21-46a4-98ed-cf10ab7b1465.png)
![image](https://user-images.githubusercontent.com/479441/160856877-9609e4ae-3ba3-449b-b4dd-fecbde960576.png)

**Comment by hjwp** [comment]
just to make sure we're on the same page, you'd like to remove non-semantically relevant parts from diffs, is that right?

you're not saying that the test reported an assert equals failure on two objects that are actually the same?  that would be very surprising.

i guess one solution would be to try and canonicalise the left and right hand sides.  can you think of a way of doing that?

**Comment by erdnaxeli** [comment]
> just to make sure we're on the same page, you'd like to remove non-semantically relevant parts from diffs, is that right?

yes, that's it. It does not make the test fail, but if the test fails those diff are reported.

**Comment by hjwp** [comment]
gotcha.  well i guess you'd need a way to "canonicalize" the whitespace and commas on both sides -- can you think of a way of doing that?

i guess you could do it manually in your tests, before the assert?
